src/lib/utils/cache.ts

```diff
@@ -4,6 +4,7 @@ import type { CacheEntry } from '$lib/utils/types';
 export const contributorsStore = writable<Map<string, CacheEntry<unknown>>>(new Map());
 
 const CACHE_TTL = 60 * 60 * 1000;
+const token = import.meta.env.VITE_GITHUB_TOKEN;
```

Comment:
Hi @grmbyrn why did you add the token again? I mean, it's fine for development, but if you add it, it should be optional for production, and the authorization header should not be included when it is empty.

Reply:
Hi @josecelano I started deleting the console logs, then undid them to check something. I think I undid deleting the token without realising before I permanently deleted the console logs.